### PR TITLE
#73 patch for CI testing with Travis 

### PR DESCRIPTION
**Pull request by jsoeterbroek**
#73 patch for CI testing with Travis.

### CODE REVIEW

**Comment by jsoeterbroek** [comment]
Please review and comment

**Comment by dirkx** [comment]
Tested, works.

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ About
 
 This site is the project website of the notification-app by the Dutch Ministry of Health.
 
+[![Build Status](https://travis-ci.org/jsoeterbroek/nl-covid19-notification-app-dashboard.svg?branch=patch-testing)](https://travis-ci.org/jsoeterbroek/nl-covid19-notification-app-dashboard)
```
Comment (bwbroersma) [inline README.md:6]: This should reference `minvws/nl-covid19-notification-app-website`

Comment (mellewynia) [reply README.md]: I have taken the liberty to update this. Looking good! Merged 👍

**Pull request merged**